nixodmlconverter/convert.py

```diff
@@ -279,7 +279,7 @@ def write_odml_doc(odmldoc, nixfile):
 
 def nixwrite(odml_doc, filename, mode='append'):
     filemode = None
-    if mode == 'append' or mode == 'overwrite metadata':
+    if ('append' or 'overwrite metadata') in mode:
```

Comment:
This wont work.  Other way around:

```suggestion
    if mode  in ('append', 'overwrite metadata') in :
```

Though `overwrite metadata` will have to change to match the cl argument above.